app/views/estimatedIncomeTax/ComplexEstimatedIncomeTaxView.scala.html

```diff
@@ -24,11 +24,12 @@
 @import uk.gov.hmrc.tai.util.constants.TaxRegionConstants._
 @import scala.concurrent.ExecutionContext
 @import views.html.estimatedIncomeTax.navigationLinks
+@import controllers.auth.AuthenticatedRequest
 
 @this(main: MainTempleteNew, loginStatus: LoginStatus)
 
 @(viewModel: uk.gov.hmrc.tai.viewModels.estimatedIncomeTax.ComplexEstimatedIncomeTaxViewModel, iFormLinks: Html)(implicit
-request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,  ec: ExecutionContext)
+request: AuthenticatedRequest[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,  ec: ExecutionContext)
```

Comment:
if we can do this, I think this is better than modifying the Models to include the fullName, thanks for the pattern!